bin/http-server

```diff
@@ -92,10 +92,11 @@ else {
   listen(port);
 }
 
+/* jshint evil:true */
 function listen(port) {
   var options = {
     root: argv._[0],
-    cache: argv.c,
+    cache: eval(argv.c),
```

Comment:
While this definitely works to pass a function through (which is good!), using `eval` runs into an issue when `argv.c` is meant to be a string like `'max-age=120'`. While that string is a perfectly valid cache header, it is not a valid JS expression, so `eval` will error.